04-single-hop-6lowpan-icmp/test_spec04.py

```diff
@@ -12,13 +12,21 @@
 
 APP = 'examples/gnrc_networking'
 TASK10_APP = 'tests/gnrc_udp'
+PKTBUF_PROBE_DELAY = 5
+PKTBUF_PROBE_ATTEMPTS = 12
 pytestmark = pytest.mark.rc_only()
 
 
 class Shell(Ifconfig, GNRCICMPv6Echo, GNRCPktbufStats):
     pass
 
 
+def check_pktbuf(nodes, timeout=60):
+    time.sleep(timeout)
```

Comment:
```suggestion
def check_pktbuf(nodes, wait=0):
    if wait:
        time.sleep(wait)
```

1. It's a wait time, not a timeout.
2. There are cases where one does not need to wait which should be the default.